<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" ignored="5" total="13" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Test-class Suite" duration-ms="223936" started-at="2020-01-22T12:37:29Z" finished-at="2020-01-22T12:41:13Z">
    <groups>
    </groups>
    <test name="Test-class test 1 Production" duration-ms="223936" started-at="2020-01-22T12:37:29Z" finished-at="2020-01-22T12:41:13Z">
      <class name="TestNG.sanityAllOptionsTest">
        <test-method status="PASS" signature="initialOfferQuestions()[pri:10, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="initialOfferQuestions" duration-ms="59058" started-at="2020-01-22T12:37:29Z" finished-at="2020-01-22T12:38:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialOfferQuestions -->
        <test-method status="FAIL" signature="initialOfferScreen()[pri:30, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="initialOfferScreen" duration-ms="98786" started-at="2020-01-22T12:38:28Z" finished-at="2020-01-22T12:40:07Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in INITIAL OFFER SCREEN QUESTION]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in INITIAL OFFER SCREEN QUESTION
at TestNG.sanityAllOptionsTest.initialOfferScreen(sanityAllOptionsTest.java:546)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialOfferScreen -->
        <test-method status="FAIL" signature="finalOfferQuestions()[pri:40, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="finalOfferQuestions" duration-ms="10005" started-at="2020-01-22T12:40:07Z" finished-at="2020-01-22T12:40:17Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.finalOfferQuestions(sanityAllOptionsTest.java:781)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finalOfferQuestions -->
        <test-method status="FAIL" signature="finalOfferScreen()[pri:50, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="finalOfferScreen" duration-ms="10006" started-at="2020-01-22T12:40:17Z" finished-at="2020-01-22T12:40:27Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.finalOfferScreen(sanityAllOptionsTest.java:843)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finalOfferScreen -->
        <test-method status="FAIL" signature="covers()[pri:70, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="covers" duration-ms="10006" started-at="2020-01-22T12:40:27Z" finished-at="2020-01-22T12:40:37Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.covers(sanityAllOptionsTest.java:919)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- covers -->
        <test-method status="FAIL" signature="payments()[pri:80, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="payments" duration-ms="13004" started-at="2020-01-22T12:40:37Z" finished-at="2020-01-22T12:40:50Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.payments(sanityAllOptionsTest.java:963)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- payments -->
        <test-method status="FAIL" signature="returnSteps()[pri:85, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="returnSteps" duration-ms="10008" started-at="2020-01-22T12:40:50Z" finished-at="2020-01-22T12:41:00Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.returnSteps(sanityAllOptionsTest.java:1091)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- returnSteps -->
        <test-method status="FAIL" signature="forwardSteps()[pri:86, instance:TestNG.sanityAllOptionsTest@2cf3d63b]" name="forwardSteps" duration-ms="13009" started-at="2020-01-22T12:41:00Z" finished-at="2020-01-22T12:41:13Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed in ScreenShot]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.testException(sanityAllOptionsTest.java:135)
at TestNG.sanityAllOptionsTest.forwardSteps(sanityAllOptionsTest.java:1200)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forwardSteps -->
      </class> <!-- TestNG.sanityAllOptionsTest -->
    </test> <!-- Test-class test 1 Production -->
  </suite> <!-- Test-class Suite -->
</testng-results>
